### ISSUE TITLE
[vite]: Rollup failed to resolve import "#minpath" from "node_modules/vfile/lib/index.js".

### ISSUE THREAD

**Issue by ahmedrowaihi** [remarkjs/react-markdown#864]
### Initial checklist

- [X] I read the [support docs](https://github.com/remarkjs/.github/blob/main/support.md)
- [X] I read the [contributing guide](https://github.com/remarkjs/.github/blob/main/contributing.md)
- [X] I agree to follow the [code of conduct](https://github.com/remarkjs/.github/blob/main/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Aremarkjs&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

react-markdown@9.0.1, vfile@6.x.x

### Link to runnable example

scaffold a react app with vite

### Steps to reproduce

- install react-markdown
- run the build
- observe error
 ```
[vite]: Rollup failed to resolve import "#minpath" from "node_modules/vfile/lib/index.js".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
error during build:
Error: [vite]: Rollup failed to resolve import "#minpath" from "node_modules/vfile/lib/index.js".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
```

### Expected behavior

it should use the `minpath.browser`, `minproc.browser`, `minurl.browser` imports

### Actual behavior

it fails to use correct imports

### Affected runtime and version

node@v22.3.0, vite@3.x.x

### Build and bundle tools

Rollup

**Comment by ChristianMurphy** [comment]
It's a bug in site's export resolution.
This was fixed several major versions ago in vite.
Upgrade your vite version.

**Comment by gabrieljablonski** [comment]
> yes, I added it for the community in case someone will hit this issue and have no option to migrate vite

and thanks for that. I'm running v3, and there's no way in hell i'm upgrading my bundler because of 3 lines of code on a dependency of a dependency 😂

**Comment by gafish** [comment]
It was solved after I downgraded the version to 7.0

**Comment by jacob2733** [comment]
If you're using webpack to build, you could solve this by add the resolve alias to your webpack config. Like 
`  '#minpath': path.resolve(root, 'node_modules/vfile/lib/minpath.browser.js'),
            '#minproc': path.resolve(root, 'node_modules/vfile/lib/minproc.browser.js'),
            '#minurl': path.resolve(root, 'node_modules/vfile/lib/minurl.browser.js'),`
